doc/build-unix.md

```diff
@@ -197,17 +197,21 @@ Berkeley DB
 -----------
 
 The legacy wallet uses Berkeley DB. To ensure backwards compatibility it is
-recommended to use Berkeley DB 4.8. If you have to build it yourself, you can
-use [the installation script included in contrib/](/contrib/install_db4.sh)
-like so:
-
-```shell
-./contrib/install_db4.sh `pwd`
+recommended to use Berkeley DB 4.8. If you have to build it yourself, and don't
+want to use any other libraries built in depends, you can do:
+```bash
+make -C depends NO_BOOST=1 NO_LIBEVENT=1 NO_QT=1 NO_SQLITE=1 NO_NATPMP=1 NO_UPNP=1 NO_ZMQ=1 NO_USDT=1
+...
+to: /path/to/bitcoin/depends/x86_64-pc-linux-gnu
 ```
+and configure using the following:
+```bash
+export BDB_PREFIX="/path/to/bitcoin/depends/x86_64-pc-linux-gnu"
 
-from the root of the repository.
-
-Otherwise, you can build Bitcoin Core from self-compiled [depends](/depends/README.md).
+./configure \
+    BDB_LIBS="-L${BDB_PREFIX}/lib -ldb_cxx-4.8" \
+    BDB_CFLAGS="-I${BDB_PREFIX}/include"
+```
```

Comment:
Actually, for an intention like _"don't want to use any other libraries built in depends"_ there is a dedicated `ALLOW_HOST_PACKAGES=1` dependencies option:

```bash
$ make -C depends NO_BOOST=1 NO_LIBEVENT=1 NO_QT=1 NO_SQLITE=1 NO_NATPMP=1 NO_UPNP=1 NO_ZMQ=1 NO_USDT=1 ALLOW_HOST_PACKAGES=1
$ ./configure CONFIG_SITE=$PWD/depends/x86_64-pc-linux-gnu/share/config.site --with-gui --with-sqlite --enable-zmq --with-miniupnpc --with-natpmp --enable-usdt
```

Reply:
> ALLOW_HOST_PACKAGES=1 dependencies option:

I don't particularly like the existence of that option in general, or want to introduce it here. So given that we acheive the same thing here (only bdb in the prefix) regardless of whether it's used, I'd rather not add it.

Reply:
That is also a somwhat more convoluted/confusing `./configure` invocation, as opposed to just passing the flags.

Reply:
> That is also a somewhat more convoluted/confusing `./configure` invocation

True.

> as opposed to just passing the flags.

... which is error-prone: if a typo slipped in `BDB_LIBS` or `BDB_CFLAGS`, the `configure` script is not testing them and only following `make` will fail.

Reply:
> which is error-prone:

That's hardly any worse than making users type out all this `./configure CONFIG_SITE=$PWD/depends/x86_64-pc-linux-gnu/share/config.site --with-gui --with-sqlite --enable-zmq --with-miniupnpc --with-natpmp --enable-usdt`, where any typo or forgotten flag would result in missing features that wouldn't be noticed until runtime.

Reply:
Users can read "Options used to compile and link:" output of the `configure` script to ensure all requested features are indeed enabled. But we cannot rely on users' reasonable behavior :)

Reply:
I agree with either approach, so labeling this comment as resolved.